docs/how-to-guides/openstack-swift.md

```diff
@@ -0,0 +1,32 @@
+Sometimes it is desirable to cache swift storage objects inside the charm for faster processing and reducing the number of requests to the [swift](https://docs.openstack.org/swift/latest/) server.
```

Comment:
Should this have a first `# <H1>`? Other wise the first line will be used as the title...